### PR TITLE
Routing home page

### PR DESCRIPTION
**Pull request by agderes**
solved the issue with scroll up to top of the page when click on the home menu ite

### CODE REVIEW

File: src/components/TheSideNavMenu.vue [hunk 1]
```diff
@@ -4,8 +4,8 @@
     <!-- LIST ITEMS CONTAINER-->
     <v-list dense nav>
       <!-- HOME -->
-      <router-link to="/home" v-slot="{ href, route, navigate, isActive, isExactActive }">
-        <v-list-item @click="navigate">
+      <router-link to= '/home' v-slot="{href, route, navigate, isActive, isExactActive }">
+        <v-list-item  @click="scrollToHome()">
```
Comment (1kh4l) [inline src/components/TheSideNavMenu.vue:8]: there's no need to use another function, try to reuse the one that handles the routing for the menu items

Comment (agderes) [reply src/components/TheSideNavMenu.vue]: please, check the file again. I solved the issue

File: src/components/TheSideNavMenu.vue [hunk 2]
```diff
@@ -4,8 +4,9 @@
     <!-- LIST ITEMS CONTAINER-->
     <v-list dense nav>
       <!-- HOME -->
-      <router-link to="/home" v-slot="{ href, route, navigate, isActive, isExactActive }">
-        <v-list-item @click="navigate">
+      <router-link to= '/home' v-slot="{href, route, navigate, isActive, isExactActive }">
+      <!-- Creates home menu without subitem -->
```
Comment (1kh4l) [inline src/components/TheSideNavMenu.vue:8]: That's nice :+1:

File: src/components/TheSideNavMenu.vue [hunk 3]
```diff
@@ -44,16 +45,20 @@
...
+import sideNavTree from '../assets/sidenav-menu.json';
 
 export default {
   name: 'SideNavMenu',
   props: {
   },
   methods: {
-    menuRouting(item, subItem) {
+    menuRouting(item, subItem = null) {
+    // Allows to go to top of the home page
```
Comment (1kh4l) [inline src/components/TheSideNavMenu.vue:56]: please align the comments with the same spacing of code

**Pull request merged**